board/piksiv3/rootfs-overlay/etc/monitrc

```diff
@@ -39,6 +39,22 @@ check process zmq_adapter_rpmsg_piksi101 with pidfile /var/run/zmq_adapter_rpmsg
     start program = "/etc/init.d/S83zmq_adapter_rpmsg_piksi101 start"
     stop program = "/etc/init.d/S83zmq_adapter_rpmsg_piksi101 stop"
 
+check process skylark_upload_daemon with pidfile /var/run/skylark_upload_daemon.pid
```

Comment:
@mfine I wonder if we should should consolidate these into a single `check process`. If the download dies and the adapters are still running, then the download restarts, writes from the download fail to get through to the fifo. I think we can manage all this by just having a single process manager fail if any of the four processes here fail. This should make for a clean start.